prow/config/config.go

```diff
@@ -31,7 +32,7 @@ import (
 
 	"github.com/ghodss/yaml"
 	"github.com/sirupsen/logrus"
-	"gopkg.in/robfig/cron.v2"
+	cron "gopkg.in/robfig/cron.v2"
```

Comment:
Saw this change in Paul's PR as well, what's up here?

Reply:
goimports in my editor (ST3) removes this import because it's used as `cron` in the code.

Reply:
Thanks, xref-ing